tableprint/printer.py

```diff
@@ -203,12 +229,14 @@ def row(values, width=None, format_spec=FMT, align=ALIGN, style=STYLE):
         A string consisting of the full row of data to print
     """
     if width is None:
-      width = max_width(values, format_spec)
+        width = max_width(values, format_spec)
 
     tablestyle = STYLES[style]
     widths = parse_width(width, len(values))
 
-    assert isinstance(format_spec, string_types) | isinstance(format_spec, list), \
+    format_spec_is_valid_type = isinstance(format_spec, string_types) or \
```

Comment:
The only thing I'm really worried about—I'm *pretty sure* that changing the pipe to an `or` is appropriate here; @nirum, can you confirm?

Reply:
should be fine